disnake/guild_scheduled_event.py

```diff
@@ -373,6 +391,12 @@ async def edit(
         - ``entity_metadata`` with a location field must be provided
         - ``scheduled_end_time`` must be provided
 
+        If updating ``channel`` to ``None``, ``entity_type`` will be set to
+        :attr:`GuildScheduledEventEntityType.external`.
+
+        If updating ``channel`` to :class:`.abc.Snowflake`, ``entity_type`` will be set to the channel's type
+        if the type is either :class:`ChannelType.voice` or :class:`ChannelType.stage_voice`.
```

Comment:
missing a versionchanged, also should update the cases when TypeError is raised.

Reply:
Looks like the `versionchanged` sections here and on `create_scheduled_event` got lost in a commit somewhere